docs/source/tutor/_hello_world.ipynb

```diff
@@ -0,0 +1,698 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hello, World\n",
+    "\n",
+    "Test the new bluesky installation with the _Hello, World!_  code in the `~/bluesky` directory.\n",
+    "\n",
+    "**Note:** the tilde (~) character represents the current user's home directory. This is a shortcut that can be used to specify file paths without having to type out the entire path to the home directory.\n",
+    "\n",
+    "<details>\n",
+    "<summary>How to install Bluesky?</summary>\n",
+    "\n",
+    "Visit <a href=https://bcda-aps.github.io/bluesky_training/instrument/_install_new_instrument.html> Bluesky Installation</a>.\n",
+    "</details>\n",
+    "\n",
+    "<details>\n",
+    "<summary>How to start a Bluesky session?</summary>\n",
+    "\n",
+    "Visit this page (TODO).\n",
+    "</details>\n",
+    "\n",
+    "\n",
+    "<details>\n",
+    "<summary>Not familiar with object-oriented programming glossary: class, object, method and instances?</summary>\n",
+    "\n",
+    "- A **class** is like a blueprint or a template that defines the characteristics and behaviors of a particular type of object. For example, we can define a `Dog` class that includes attributes such as breed, age, and name, as well as behaviors such as barking and wagging its tail.\n",
+    "\n",
+    "- An **object** is an **instance** of a class. So, if we have a `Dog` class, we can create objects of that class, such as \"Fido\" and \"Buddy\". Each object of the `Dog` class will have its own set of **attributes**, such as \"Fido\" being a Golden Retriever and \"Buddy\" being a Chihuahua, and **methods**, such as \"Fido\" barking loudly and \"Buddy\" wagging its tail.\n",
+    "\n",
+    "- An **instance** is a specific occurrence of an **object** created from a **class**. For example, with our `Dog` class, we can create an instance of that class called \"my_dog\" with specific attributes and behaviors.\n",
+    "\n",
+    "- A **method** is a function that is defined in a **class** and can be called on an **object** of that class. For example, the \"bark\" method defined in the `Dog` class can be called on the object \"my_dog\" to make it bark.\n",
+    "</details>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Follow these steps to load the `bluesky` and `databroker` packages for data collection activities."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pathlib, sys, os\n",
+    "my_path = str(pathlib.Path.home() / \"bluesky\")\n",
```

Comment:
Keep `my_path` as a pathlib object.  Maybe someone will use it later.  Change to:

```py
my_path = pathlib.Path.home() / "bluesky"
sys.path.append(str(my_path))
```

Convert to `str` when necessary.